libshaderc/include/shaderc.h

```diff
@@ -252,56 +251,59 @@ typedef struct shaderc_spv_module* shaderc_spv_module_t;
 // The input_file_name is a null-termintated string. It is used as a tag to
 // identify the source string in cases like emitting error messages. It
 // doesn't have to be a 'file name'.
-// By default the source string will be compiled into SPIR-V binary
-// and a shaderc_spv_module will be returned to hold the results of the
-// compilation. When disassembly mode or preprocessing only mode is enabled
-// in the additional_options, the source string will be compiled into char
-// strings and held by the returned shaderc_spv_module.  The entry_point_name
-// null-terminated string defines the name of the entry point to associate
-// with this GLSL source. If the additional_options parameter is not NULL,
-// then the compilation is modified by any options present. May be safely
-// called from multiple threads without explicit synchronization. If there
-// was failure in allocating the compiler object NULL will be returned.
-shaderc_spv_module_t shaderc_compile_into_spv(
+// By default the source string will be compiled into SPIR-V binary and a
+// shaderc_compilation_result will be returned to hold the results of the
+// compilation. When disassembly mode or preprocessing only mode is enabled in
+// the additional_options, the source string will be compiled into char strings
+// and held by the returned shaderc_compilation_result. The entry_point_name
+// null-terminated string defines the name of the entry point to associate with
+// this GLSL source. If the additional_options parameter is not NULL, then the
+// compilation is modified by any options present. May be safely called from
+// multiple threads without explicit synchronization. If there was failure in
+// allocating the compiler object NULL will be returned.
+shaderc_compilation_result_t shaderc_compile_into_spv(
     const shaderc_compiler_t compiler, const char* source_text,
     size_t source_text_size, shaderc_shader_kind shader_kind,
     const char* input_file_name, const char* entry_point_name,
     const shaderc_compile_options_t additional_options);
 
-// The following functions, operating on shaderc_spv_module_t objects, offer
-// only the basic thread-safety guarantee.
+// The following functions, operating on shaderc_compilation_result_t objects,
+// offer only the basic thread-safety guarantee.
 
-// Releases the resources held by module.  It is invalid to use module for any
-// further operations.
-void shaderc_module_release(shaderc_spv_module_t module);
+// Releases the resources held by the result object. It is invalid to use the
+// the data held by the result object for any further operations.
```

Comment:
Typo: "the the". Also, I'd remove the entire phrase "the data held by" and keep just "the result object".